### PR TITLE
ovirt e2e: add machineNetwork to install-config.yaml

### PR DESCRIPTION
**Pull request by eslutsky**
it is required for the bootstrap  IPV4 selection/filtering,
this  changed is caused  by https://github.com/openshift/cluster-etcd-operator/pull/384

### CODE REVIEW

File: ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml [hunk 1]
```diff
@@ -412,8 +412,10 @@ objects:
                 # 31 is used to trigger the instance customization (the disk size is 16 Gi)
                 sizeGB: 31
               vmType: server
-              instanceTypeID: 
+              instanceTypeID:
```
Comment (rgolangh) [inline ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml:415]: please remove this line

File: ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml [hunk 2]
```diff
@@ -442,6 +444,8 @@ objects:
           ${SSH_PUB_KEY}
         EOF
 
+        cp /tmp/artifacts/installer/install-config.yaml /tmp/artifacts/installer/install-config-bk.yaml
```
Comment (rgolangh) [inline ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml:447]: can you leave a comment why is this needed?

Comment (Gal-Zaidman) [reply ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml]: I think it was only for testing, we shouldn't save it

Comment (eslutsky) [reply ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml]: correct, it was for testing only, removed

**Comment by ironcladlou** [comment]
Interesting — my understanding was that machineCIDR was required (cc @openshift/installer @hexfusion @retroflexer) but apparently not?

File: ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml [hunk 3]
```diff
@@ -414,6 +414,8 @@ objects:
               vmType: server
               instanceTypeID: 
           replicas: 2
+        networking:
+          machineCIDR: 192.168.0.0/16
```
Comment (ironcladlou) [inline ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml:418]: This field is deprecated; I think you should be using `networking.machineNetwork`

**Comment by abhinavdahiya** [comment]
Don't use the deprecated field. Should use the machineNetwork slice for configuring this.

Never if the networking fields are required to be set by the user because we pick a default for these when not specified.. therefore consumers can expect these to be set ...

File: ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml (hunk 3 above)
Comment (eslutsky) [reply ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml]: done, let's see if this pass  the installation.

**Comment by ironcladlou** [comment]
Although I think it's good and we should specify the machine CIDR, I also think that our new IP discovery should fall back to the old behavior if all else fails, which I think would have kept the current (and possibly one other known) scenario working without any changes. I opened https://github.com/openshift/cluster-etcd-operator/pull/388 which should make the CIDR change here unnecessary, although I still think we should explicitly specify the CIDR rather than letting the platform guess when possible. I would like to see if the patch in https://github.com/openshift/cluster-etcd-operator/pull/388 passes the existing job without this change before merging, if possible...

/hold
